### ISSUE TITLE
BLE Central on Android?

### ISSUE THREAD

**Issue by JohnHigbie** [don/pgb-test#1]
Hi Don. Thanks for posting this. Got the app working ok on iOS, but on Android Lollipop, which supposedly supports BLE Central, the Red Bear BLE Mini does not show up when I hit "Refresh." I am able to get the BLE Mini to pair from the Bluetooth settings. I've got the latest fw in the Mini and the latest Lollipop. Have you tested this on an Android device? -John

**Comment by don** [comment]
@JohnHigbie This works on my Nexus 5 running Lollipop with the RedBearLab BLE shield. It should work fine with the BLEMini or any other RBL boards.

You shouldn't pair the BLEMini with your phone, the app will discover BLE devices that are advertising. 

I suspect that after you paired with the BLEMini your phone connected to it. Normally when a client connects to a peripheral, the peripheral stops advertising and it won't show up when the app tries to discover it.

Try deleting the pairing and run the app again.

**Comment by JohnHigbie** [comment]
Don, actually I had tried it both paired and unpaired. Perhaps there's a difference between the shield and the Mini?

**Comment by don** [comment]
I'll try with a BLEmini later today

**Comment by don** [comment]
@JohnHigbie the code worked OK with my BLEMini.and Nexus 5

Have you updated the firmware on the BLEMini? Try Biscuit-UART_20140409.bin from https://github.com/RedBearLab/Biscuit/tree/master/release

**Comment by JohnHigbie** [comment]
Yes, that's the version I was using. However, I just re-installed the app on the Nexus 7, with no changes, and now it WORKS. Don't know why, but if you put a gun to my head for a theory, maybe because I had previously installed the app before updating to Lollipop, instead of after? Anyway Don, thanks for sticking with me and doing all this work. -John
